src/main/java/uk/gov/hmcts/reform/civil/model/CaseData.java

```diff
@@ -697,4 +702,56 @@ public YesOrNo getRespondent2Represented() {
             .filter(Objects::nonNull)
             .findFirst().orElse(null);
     }
+
+    @JsonIgnore
+    public DefendantResponseStatus getDefendantResponseStatus() {
+        if (defenceAdmitPartPaymentTimeRouteRequired != null
+            && defenceAdmitPartPaymentTimeRouteRequired != RespondentResponsePartAdmissionPaymentTimeLRspec.IMMEDIATELY) {
+            return DefendantResponseStatus.ELIGIBLE_FOR_CCJ;
+        }
+        if (respondent1ClaimResponsePaymentAdmissionForSpec == RespondentResponseTypeSpecPaidStatus.PAID_FULL_OR_MORE_THAN_CLAIMED_AMOUNT
+            && respondent1DQ.getResponseClaimCourtLocationRequired() != null) {
+            return DefendantResponseStatus.CLAIMANT_ACCEPTED_STATES_PAID;
+        }
+        if (respondent1ClaimResponsePaymentAdmissionForSpec == RespondentResponseTypeSpecPaidStatus.PAID_FULL_OR_MORE_THAN_CLAIMED_AMOUNT) {
```

Comment:
This if statement can be merged with the previous one, which should improve readability. E.g.:

       
`if (respondent1ClaimResponsePaymentAdmissionForSpec == RespondentResponseTypeSpecPaidStatus.PAID_FULL_OR_MORE_THAN_CLAIMED_AMOUNT) {`
`    if (respondent1DQ.getResponseClaimCourtLocationRequired() != null) {`
`        return DefendantResponseStatus.CLAIMANT_ACCEPTED_STATES_PAID;`
`    }`
`    return DefendantResponseStatus.PAID_IN_FULL;`
`}`